k8s-publishing-bot.md

```diff
@@ -1,15 +1,23 @@
 # Kubernetes Publishing Bot
 
 The publishing-bot for the Kubernetes project is running in the
-`k8s-publishing-bot` namespace  on a CNCF sponsored GKE cluster
-`development2` in the `kubernetes-public` project.
+`publishing-bot` namespace  on a CNCF sponsored GKE cluster
+`aaa` in the `kubernetes-public` project.
 
 The bot is responsible for updating `go.mod`/`Godeps` for target repos.
 
 ## Permissions
 
-The cluster can be accessed by members of the [k8s-infra-cluster-admins]
-google group. Members can be added to the group by updating [groups.yaml].
+If you need access to any of the following, please update [groups.yaml].
+
+### Cluster
+
+The cluster can be accessed by [k8s-infra-cluster-admins@kubernetes.io].
```

Comment:
Members of [k8s-infra-rbac-publishing-bot@kubernetes.io](https://groups.google.com/a/kubernetes.io/forum/#!forum/k8s-infra-rbac-publishing-bot) can access to the cluster. They are only authorized to operate in the namespace `publishing-bot`.
https://github.com/kubernetes/k8s.io/blob/master/groups/groups.yaml#L405-L418

Reply:
Fixed :+1: